components/signup/signup_controller/signup_controller.jsx

```diff
@@ -236,6 +236,13 @@ export default class SignupController extends React.Component {
         }
 
         if (this.props.isLicensed && this.props.enableLDAP) {
+            let query = '';
```

Comment:
Perhaps the following instead?
```
(new URLSearchParams(this.props.location.search)).append('extra', 'create_ldap').toString()
```

(`toString()` being redundant if you're just going to concatenate with `/login`.)

Reply:
Ah yes, I was looking for a better way to do that. I'll update